### PR TITLE
Allow passing array to AssociationCollection::type().

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

**Comment by markstory** [comment]
Seems reasonable to me.

File: src/ORM/AssociationCollection.php [hunk 1]
```diff
@@ -111,14 +111,17 @@ public function keys()
...
      * @return array An array of Association objects.
      */
     public function type($class)
     {
+        $class = (array)$class;
+
         $out = array_filter($this->_items, function ($assoc) use ($class) {
             list(, $name) = namespaceSplit(get_class($assoc));
-            return $class === $name;
+            return in_array($name, $class);
```
Comment (lorenzo) [inline src/ORM/AssociationCollection.php]: pass last argument as `true` for strict check

**Comment by ADmad** [comment]
@lorenzo Done

**Comment by lorenzo** [comment]
:clap:

**Pull request merged**